specification/schemas/ServiceRateImport.json

```diff
@@ -0,0 +1,64 @@
+{
+  "type": "object",
+  "required": [
+    "type"
+  ],
+  "properties": {
+    "type": {
+      "type": "string",
+      "pattern": "^service-rate-imports$",
+      "example": "service-rate-imports"
```

Comment:
```suggestion
      "pattern": "^rate-imports$",
      "example": "rate-imports"
```

Reply:
Here the plural on imports is ok because I think `rates-imports` is not exactly gramatically correct.